mangoPURE/models/utils.py

```diff
@@ -0,0 +1,102 @@
+from transformers import WhisperFeatureExtractor
+import torch
+from typing import List, Dict
+from ..data.base import SegmentInfo, MixedExample
+from ..globals import RATE
+
+
+class Whisper:
+
+    @staticmethod
+    def update_segment_infos(audio: torch.Tensor, infos: List[SegmentInfo]):
+        """
+        Transforms segment infos according to whisper feature extractor output
+        !!!updates info in-place
+        """
+        num_secs = audio.shape[0] / RATE
+        if num_secs > 30.0:
```

Comment:
Bad

Reply:
better pass this value somewhere and not hardcode

Reply:
okay